vcpkg/consume/manifest-mode.md

```diff
@@ -0,0 +1,321 @@
+---
+title: "Tutorial: Install a dependency from a manifest file"
+description: Learn to install your projects dependencies with vcpkg by using a manifest file.
+author: vicroms
+ms.author: viromer
+ms.date: 8/10/2023
+ms.prod: vcpkg
+ms.topic: tutorial
+---
+# Tutorial: Install a dependency from a manifest file
+
+vcpkg has two operation modes: classic mode, and manifest mode. This article describes how to install packages using manifest mode, which is the recommended workflow for most users.
```

Comment:
Should we consider adding a zone group for manifest vs. classic?

Reply:
I like having the separate articles in this case, it is not a clear cut like it is for OS selection